DiceKeys/Core/Api/AuthenticationTokens.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  AuthenticationTokens.swift
+//  DiceKeys
+//
+//  Created by Stuart Schechter on 2021/02/04.
+//
+
+import Foundation
+
+class AuthenticationTokens {
+    static var tokenToUrl: [String: String] = [:]
+
+    static func setToken(authToken: String, replyToUrl: String) {
```

Comment:
Where are we using this function to set the token? Is it not in this PR?

Reply:
Not in this PR.  Will use later.